api/config/test_institutions.json

```diff
@@ -0,0 +1,19 @@
+[{
```

Comment:
File naming nit: for other configs we have {configType}_{env}.json. Prefer that convension here, e.g. institutions_test.json